pkg/config/config.go

```diff
@@ -712,6 +712,7 @@ func InitConfig(config Config) {
 	// It may be useful to increase it when logs writing is slowed down, that
 	// could happen while serializing large objects on log lines.
 	config.BindEnvAndSetDefault("logs_config.aggregation_timeout", 1000)
+	config.BindEnvAndSetDefault("logs_config.file_scan_period", 10)
```

Comment:
Let's mention the unit (seconds) in a comment.

More importantly, since your code expects this to be a `float64`, please define the default value as a float so that the implicit type casting that viper sometimes uses is consistent with the type the code expects (otherwise viper may cast the configured value to an int).